doc/source/whatsnew/v0.22.0.txt

```diff
@@ -89,6 +89,7 @@ Bug Fixes
 
 - Bug in ``pd.read_msgpack()`` with a non existent file is passed in Python 2 (:issue:`15296`)
 - Bug in ``DataFrame.groupby`` where key as tuple in a ``MultiIndex`` were interpreted as a list of keys (:issue:`17979`)
+- Bug in ``pd.read_csv`` where a multi-index with duplicate columns was not being mangled appropriately (:issue: `18062`)
```

Comment:
Can you make the following changes: 
  - <code>\`\`pd.read_csv\`\`</code> -> <code>:func:\`read_csv\`</code>
  - "multi-index" -> <code>\`\`MultiIndex\`\`</code>
  - Remove the space between `:issue:` and the issue number